File: src/TrainingBundle/Entity/Animal.php
<?php

namespace TrainingBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Serializer\Annotation\Groups;
use Symfony\Component\Validator\Constraints as Assert;


/**
 * Animal
 *
 * @ORM\Table(name="animal")
 * @ORM\Entity(repositoryClass="TrainingBundle\Repository\AnimalRepository")
 */
class Animal
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;


    /**
     * @var int
     *
     * @ORM\Column(name="debutSaison", type="integer")
     */
    private $debutSaison;


    /**
     * @var int
     *
     * @ORM\Column(name="finSaison", type="integer")
     */
    private $finSaison;

    /**
     * @return int
     */
    public function getDebutSaison()
    {
        return $this->debutSaison;
    }

    /**
     * @param int $debutSaison
     */
    public function setDebutSaison($debutSaison)
    {
        $this->debutSaison = $debutSaison;
    }

    /**
     * @return int
     */
    public function getFinSaison()
    {
        return $this->finSaison;
    }

    /**
     * @param int $finSaison
     */
    public function setFinSaison($finSaison)
    {
        $this->finSaison = $finSaison;
    }



    /**
     * @var string
     *
     * @ORM\Column(name="categorie", type="string", length=255)
     */
    private $categorie;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=255)
     * @Assert\NotBlank(message="lieu ne doit pas etre vide")
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="text")
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="image_animal", type="string", length=255)
     */
    private $image_animal;




    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }


    /**
     * Set categorie
     *
     * @param string $categorie
     *
     * @return Animal
     */
    public function setCategorie($categorie)
    {
        $this->categorie = $categorie;

        return $this;
    }

    /**
     * Get categorie
     *
     * @return string
     */
    public function getCategorie()
    {
        return $this->categorie;
    }

    /**
     * Set nom
     *
     * @param string $nom
     *
     * @return Animal
     */
    public function setNom($nom)
    {
        $this->nom = $nom;

        return $this;
    }

    /**
     * Get nom
     *
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Animal
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @return string
     */
    public function getImageAnimal()
    {
        return $this->image_animal;
    }

    /**
     * @param string $image_animal
     */
    public function setImageAnimal($image_animal)
    {
        $this->image_animal = $image_animal;
    }







}

File: web/java/huntkingdom/src/Entities/Animal.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

/**
 *
 * @author tibh
 */
public class Animal {
   private int id;
   private String categorie ;
   private String nom;
   private String description;
   private String image_animal;
   private int debutSaison;
   private int finSaison;

    public Animal() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCategorie() {
        return categorie;
    }

    public void setCategorie(String categorie) {
        this.categorie = categorie;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getImage_animal() {
        return image_animal;
    }

    public void setImage_animal(String image_animal) {
        this.image_animal = image_animal;
    }

    public int getDebutSaison() {
        return debutSaison;
    }

    public void setDebutSaison(int debutSaison) {
        this.debutSaison = debutSaison;
    }

    public int getFinSaison() {
        return finSaison;
    }

    public void setFinSaison(int finSaison) {
        this.finSaison = finSaison;
    }

    public Animal(int id, String categorie, String nom, String description, String image_animal, int debutSaison, int finSaison) {
        this.id = id;
        this.categorie = categorie;
        this.nom = nom;
        this.description = description;
        this.image_animal = image_animal;
        this.debutSaison = debutSaison;
        this.finSaison = finSaison;
    }
    public Animal( String categorie, String nom, String description, String image_animal, int debutSaison, int finSaison) {
       
        this.categorie = categorie;
        this.nom = nom;
        this.description = description;
        this.image_animal = image_animal;
        this.debutSaison = debutSaison;
        this.finSaison = finSaison;
    }
    
     public Animal(String nom) {
        
        this.nom = nom;
        
    }

    @Override
    public String toString() {
        return "Animal{" + "id=" + id + ", categorie=" + categorie + ", nom=" + nom + ", description=" + description + ", image_animal=" + image_animal + ", debutSaison=" + debutSaison + ", finSaison=" + finSaison + '}';
    }
    public String toStringName() {
        return "Nom{" +  nom+"\n"+'}';
    }
   
   
}
